dev/oracle/21-slim/wait-until-healthy.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved
+
+#!/usr/bin/env bash
+
+if [ "$#" -ne 1 ]; then
+  >&2 echo "Please provide the container name or hash"
+  exit 1
+fi
+
+for _ in {1..50}
+do
+  state=$(docker inspect -f '{{ .State.Health.Status }}' $1 2>&1)
+  return_code=$?
+  if [ ${return_code} -eq 0 ] && [ "$state" == "healthy" ]; then
+    echo "$1 is healthy!"
+    sleep 60
```

Comment:
Why do we sleep another minute if things are healthy?

Reply:
The server processes are still initializing , small delay is needed before client can talk to DB, hence we seperated this health check script, wait-until-healthy.sh

Reply:
alright